christfellowshipchurch/web-app-v2
# Content Blocks - Adding a Video to the Layout "Content to the Right" or "Content to the Left"

### About
This PR changes the width of a video when content is to one of the sides (in page builder pages?) in the content block file in the ui-kit folder. The video used to take more space than there was and create margin on the side and the user would have to scroll to the side to see the full video, now the video is smaller (cropped) and can be seen fully and no margin is created. 

### Test Instructions
1. Go to the Testing Content Blocks Item on Rock ( [here](https://rock.christfellowship.church/page/4212?ContentItemId=14407) ) and go into the last child item (Don't stay stuck any longer) and change the content layout from "Content above an image" to Content to the left of an image or Content to the right of an image" in order to test this changes.
2. Go to [/testing-content](https://www.christfellowship.church/testing-content) on the website and see how it used to be.
3. Go to [/testing-content](https://web-app-v2-git-content-blocks-a-022b7a-christ-fellowship-church.vercel.app/testing-content) on the testing link to see how it is with the changes.

### Screenshot
|How it used to be|With the changes|
|--|--|
|<img width="1268" alt="Screenshot 2023-06-30 at 4 06 57 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/e9059422-9a18-4a84-8254-c7e924aa25f0">|<img width="1194" alt="Screenshot 2023-06-30 at 4 07 33 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/cfd9b292-e20c-4eee-aedb-a182ce81be9b">|




### Closes Tickets
[CFDP-2625](https://christfellowshipchurch.atlassian.net/browse/CFDP-2625)


[CFDP-2625]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
